app.json

```diff
@@ -3,7 +3,6 @@
     "name": "Smart Village App",
     "slug": "smart-village-app",
     "owner": "ikusei",
-    "privacy": "unlisted",
     "description": "An open source React Native app for smart villages merging regional offers and services in one place - in your pocket.",
```

Comment:
Have you tried using other values instead of deleting privacy? Like `hidden`, for example.

Reply:
no I have not.

Reply:
but it's already added back, therefore not an issue anymore